configure_clang

```diff
@@ -2,7 +2,6 @@
 export CC="clang"
 export AR="llvm-ar"
 export NM="llvm-nm"
-export LD="llvm-link"
```

Comment:
well, llvm-link seem to be buggy on my machine, but clang fail to link examples with ld, so I can't use clang with lld on my machine, but clang travis need lld, so my solution is to put lld in travis file, and let the user choose his compiler